CONTRIBUTING.md

```diff
@@ -1,120 +1,116 @@
-# Contributing Guidelines
+# Contributing to Textual
 
-🎉 **First of all, thanks for taking the time to contribute!** 🎉
+First of all, thanks for taking the time to contribute to Textual!
 
-## 🤔 How can I contribute?
+## How can I contribute?
 
-**1.** Fix issue
+You can contribute to Textual in many ways:
 
-**2.** Report bug
+ 1. [Report a bug](https://github.com/textualize/textual/issues/new?title=%5BBUG%5D%20short%20bug%20description&template=bug_report.md)
```

Comment:
Should we not cover adding a feature? Not all work is necessarily fixing something.

Reply:
I added “propose a new feature” and reworded the “fix a previously opened issue” to “work on a previously opened issue”.